modcc/astmanip.hpp

```diff
@@ -10,19 +10,30 @@
 
 // Create new local variable symbol and local declaration expression in current scope.
```

Comment:
Given the number of lines involved, I personally prefer to keep the one comment covering the return type and function declaration; otherwise it becomes a small nest of inter-referential comments.